frontend/src/metabase/query_builder/components/expressions/ExpressionMode.js

```diff
@@ -0,0 +1,50 @@
+import "brace/mode/java";
+
+class ExpressionHighlight extends window.ace.acequire(
+  "ace/mode/text_highlight_rules",
+).TextHighlightRules {
+  constructor() {
+    super();
+
+    this.$rules = {
+      start: [
+        {
+          token: "string",
+          regex: '".*?"',
+        },
+        {
+          token: "constant.numeric",
+          regex: "[+-]?\\d+(?:(?:\\.\\d*)?(?:[eE][+-]?\\d+)?)?\\b",
+        },
+        {
+          token: "variable",
+          regex: "[[a-zA-Z0-9_s→]+\\]",
```

Comment:
This makes me feel like it doesn't like other languages.
But it also doesn't like spaces, or dashes.
![image](https://user-images.githubusercontent.com/1447303/144909327-930ad342-cd24-4dd3-903f-1671cefee3ad.png)

Reply:
I've tweaked this one, so anything between the square brackets should be recognized properly.